index.html

```diff
@@ -0,0 +1,450 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Youtube Page</title>
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+	 crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="./assets/stylesheet/style.css">
+
+</head>
+
+<body>
+	<header>
+		<div class="nav-bar">
+			<section class="header-left">
+				<i class="fas fa-bars"></i>
+				<img class="logo" src="./assets/media/images/youtube.png" alt="logo">
+			</section>
+			<section class="header-center">
+				<input type="text" class="search-field" placeholder="Search">
+				<button class="search-button">
+					<i class="fas fa-search"></i>
+				</button>
+			</section>
+			<section class="header-right">
+				<i class="fas fa-video"></i>
+				<i class="fas fa-th"></i>
+				<i class="fas fa-comment"></i>
+				<i class="fas fa-bell"></i>
+				<img class="user" src="./assets/media/images/user.png" alt="user">
+			</section>
+		</div>
+
+	</header>
+	<main>
+		<section class="main-video">
+
+			<iframe width="560" height="450" src="https://www.youtube.com/embed/CvJG4sQhzsw" frameborder="0" allow="autoplay; encrypted-media"
```

Comment:
The frameborder attribute on the iframe element is obsolete. Use CSS instead

Reply:
Frameborder attribute has been removed

Reply:
👍

Reply:
Thanks